CODEOWNERS

```diff
@@ -3,4 +3,5 @@
 
 # Define designers as code owners for screenshots to automatically
 # request them as reviewer if visual changes exist
-/packages/sit-onyx/playwright/snapshots/ @jannick-ux @mj-hof
+## TODO re-enable the next line once onyx leaves the alpha stage.
+# /packages/sit-onyx/playwright/snapshots/ @jannick-ux @mj-hof
```

Comment:
Would just remove it completely. Otherwise, we will have this TODO open for long.

Reply:
OK :) I created https://github.com/SchwarzIT/onyx/issues/404 so we can find it again when the time comes :)